### ISSUE TITLE
Improve Wedge3D documentation of 'halves_to_use'.

### ISSUE THREAD

**Issue by markscheel** [sxs-collaboration/spectre#795]
In Wedge3D there is a parameter `halves_to_use` passed into the constructor,
and a corresponding enum class `WedgeHalves`.  The documentation says that this parameter determines whether to "use" logical coordinates xi in [0,1] or [-1,0], but I don't know what "use" means, and this documentation does not make it clear what this parameter does.  In particular:

  - Does this apply only for the xi direction, and not the eta direction?  (this seems to be the case).
  - Do the logical coordinates xi still go from [-1,1] if `halves_to_use` is not 'Both'? In other words, if I had an arbitrary point inside the domain and I inverse-mapped it through Wedge3D, do I get a logical point that is in [-1,1] (as opposed to [-1,0] or [0,1])? (This seems to be the case from looking through the code, but this seems to contradict the documentation).
  - It seems (from looking through the code) that if this parameter is not 'Both', the resulting shape has a face normal to the x direction (assuming default OrientationMap).  Is this the case?  It should be described in the documentation.

### Component:

- [ ] Code
- [x] Documentation
- [ ] Build system
- [ ] Continuous integration

### Desired feature:

- [ ] Detail 1
- [ ] Detail 2
- [ ] Detail 3

**Comment by mar-celine** [comment]
The logical coordinates go from -1 to 1 in both cases (using only a half or using both halves).

The statement about using the logical coordinates in [-1, 0] or [0, 1] come from the fact that the map for the half wedge is obtained by taking the interval [-1, 1] in the logical cube and then applying an intermediate map that takes points in [-1, 1] and returns points in [-1, 0] (in the case of the lower half, [0. 1] in the case of the upper half). Points in the half-interval are then passed to the *full* Wedge3D map. Since only half of the logical interval is used, the coordinate map ends up being only half of the full wedge. This is all done internally when using a half wedge; the user still provides points in [-1,1] and the inverse still returns points in [-1,1] even in the case of half wedges.

In other words, the logical coordinates will always go from [-1, 1], any references to [-1, 0] or [0, 1] refer to implementation details about how the points are computed. I should clarify this in the documentation so thanks for pointing that out!

To answer your other questions, yes, only the splitting only cuts perpendicular to the logical xi axis, and for a default orientation this will result in a shape that has a normal to the x-direction.

**Comment by kidder** [comment]
@gsb76 is this being done, done, or going to be done?
